src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -93,6 +96,14 @@ private CloudSdk(Path sdkPath, String appCommandMetricsEnvironment,
     this.processRunner = new DefaultProcessRunner(async, stdOutLineListeners, stdErrLineListeners,
         exitListeners, startListeners);
 
+    // Populate jar locations.
+    // TODO(joaomartins): Consider case where SDK doesn't contain these jars. Only App Engine
+    // SDK does.
+    JAR_LOCATIONS.put("servlet-api.jar",
```

Comment:
I know this has been submitted, but there is no guarantee these jars will exist in the future in the cloud SDK. What is the need for these?

This CL seems to be modifying more than is intended in it's title.